content/reference/user/organization/index.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
organization-admin-user.png =>

Firstname: John
Lastname: Doe
GitHub username: `jdoe` or `john.doe`

"My Org" is the name of... the organization, not a user

(Note: John/Jane Doe are the names given to anonymous corpses or to amnesiacs)

Reply:
Done